src/content/en/progressive-web-apps/_index.yaml

```diff
@@ -93,9 +93,9 @@ landing_page:
         <p>
           The Web App Manifest allows you to control how your app appears and
           how it's launched. You can specify icons for the home screen and
-          splash screen which is shown while the app is loading. Which page
-          is loaded when the app is launched, screen orientation, even
-          whether to show the browser chrome or not.
+          the splash screen shown while the app loads, which page to load when
+          the app launches, screen orientation, even whether or not to show the
+          browser chrome.
         </p>
```

Comment:
That's a step in the right direction, but it could still be better. How about:

You can specify icons for the home screen, the splash screen (shown while the app is loading), the page loaded when the app is launched, screen orientation, and even, whether or not to show the browser chrome.